common/remote/rmtssh.cpp

```diff
@@ -99,7 +99,8 @@ class CFRunSSH: public CInterface, implements IFRunSSH
                     break;
                 case 's': { // ssh params
                         bool usepssh = !password.isEmpty();
-                        cmdbuf.appendf("%s -o LogLevel=QUIET -o StrictHostKeyChecking=%s -o BatchMode=yes ",usepssh?"pssh":"ssh",strict?"yes":"no");
+                        // reset LD_LIBRARY_PATH here so ssh cmd itself doesn't use HPCC libssl/crypto as they may be different
+                        cmdbuf.appendf("%s -o LogLevel=QUIET -o StrictHostKeyChecking=%s -o BatchMode=yes ",usepssh?"pssh":"LD_LIBRARY_PATH=: ssh",strict?"yes":"no");
```

Comment:
looks like it should go infront of "pssh" too .. ?

Reply:
could use IPipeProcess::setEnv("LD_LIBRARY_PATH", <value>) instead (after call to createPipeProcess())

Reply:
There's another instance of using "ssh" (and "pssh") on line ~ 398.

Using IPipeProcess::setEnv approach would cover all cases.

Reply:
ok, thanks.  I will change to this.